packages/datagateway-search/src/table/cellRenderers.tsx

```diff
@@ -0,0 +1,67 @@
+import {
+  type Dataset,
+  formatCountOrSize,
+  type Investigation,
+  useDatasetsDatafileCount,
+  useDatasetSizes,
+  useInvestigationDatasetCount,
+  useInvestigationSize,
+} from 'datagateway-common';
+import React from 'react';
+
+/**
+ * A cell in {@link InvestigationSearchTable} that displays the count of a dataset.
+ * @param investigation
+ */
+function InvestigationDatasetCountCell({
```

Comment:
A slight concern I have is over the performance of each size/count query being separate as that has caused performance issues in the past. However, I expct Jounaid's size/count refactor to go into the same release as this search stuff so no major concern - although that's something to keep in mind for later as there's still merge jobs to do.

Reply:
I think I added this was because the batch query approach would also query for sizes that are not currently visible in the table, so when they would error out it would cause a flood of errors in the console, completely bogging down the website. Separating the query into its own component has the benefit of preventing unnecessary queries.

With the new investigation size/count refactor though, like you said, I don